### PR TITLE
User guide/outlook

### PR DESCRIPTION
**Pull request by nnenasheva**

### CODE REVIEW

**Comment by artml** [comment]
I propose the following fixes:

OroCRM enterprise version → OroCRM Enterprise edition
Preconditions → Supported versions or Requirements

System Configuration --> SYSTEM CONFIGURATION --> MS Outlook settings
_change to_
System → Configuration → Integrations →MS Outlook settings

You can choose sync direction and intervals → You may configure
set of entities synchronized → set of entities to be synchronized

Initially, default settings are applied. Uncheck "Use Default" box to change the settings.
_replace with_
To change default settings, uncheck the "Use Default" box first.
_and do not start a new paragraph._

the default values will be applied regardless → regardless of
changed in the both Outlook and CRM → in both
alerts (if any) will pop-up → pop up _(because verb, not noun)_
the Last Name value will be used for the both first name and last name → is used for both first name and last name _(twice)_ _(if we use present tense before the table ("are mapped"), we should use the same tense everywhere)_
Any number of Emails may be mapped → All existing emails are mapped 

Outlook status not available → statuses
_and please align the column header to the left_

Only calendar assigned to the specified user → the calendar

_On the sync flowchart: questions should be questions, e.g. "Is there an Outlook record with the matching OroCRM_id?"_

_Tense in the If–Then table should be consistent: "Values [...] replace" but "OroCRM will update"_

**Comment by DimaSoroka** [comment]
@nnenasheva please apply fixes requested by @artml

File: user_guide/outlook_sync.rst [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+
+Settings on the Outlook Side
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have installed the .msi file, *"OroCRM for Outlook*" menu will appear in your Oultlook menu bar. 
+
+.. image:: ./img/outlook/outlook_menu_bar.png
+
+Choose "Settings". *"OroCRM for Outlook Settings"* window will emerge.  The following fields are available
+
```
Comment (DimaSoroka) [inline user_guide/outlook_sync.rst]: Please add user_guide/img/outlook/outlook_settings.png here.

File: user_guide/outlook_sync.rst [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+.. csv-table::
+  :header: "**Name**","**Description**" 
+  :widths: 10, 30
+
+  "**OroCRM URL**","The Web address of your OroCRM instance. Mandatory field."
+  "**Ignore self-signed certificate**","The box must be checked. Mandatory field."
```
Comment (DimaSoroka) [inline user_guide/outlook_sync.rst]: Please explain the meaning of this field

**Pull request merged**